### PR TITLE
Update passport to version 7.0

### PR DESCRIPTION
**Pull request by diegofelix**

### CODE REVIEW

File: src/Client.php [hunk 1]
```diff
@@ -12,6 +12,41 @@ class Client extends Model
...
+    ];
+
+    /**
+     * Get the user that the client belongs to.
+     *
+     * @return \Illuminate\Database\Eloquent\Relations\BelongsTo
+     */
+    public function user()
+    {
+        return $this->belongsTo(
```
Comment (vitorbari) [inline src/Client.php]: Use mongolid method here

File: src/Console/ClientCommand.php [hunk 2]
```diff
@@ -89,8 +86,29 @@ protected function createPasswordClient(ClientRepository $clients)
...
+        $this->outputClientDetails($client);
+    }
+
+    /**
+     * Create a client credentials grant client.
+     *
+     * @param  \Laravel\Passport\ClientRepository  $clients
+     * @return void
+     */
+    protected function createClientCredentialsClient(ClientRepository $clients)
```
Comment (vitorbari) [inline src/Console/ClientCommand.php:99]: The `$allowedScopes` steps should be moved into this method.

**Review by vitorbari — CHANGES_REQUESTED** [review]

File: src/Client.php (hunk 1 above)
Comment (diegofelix) [reply src/Client.php]: Done

File: src/Console/ClientCommand.php (hunk 2 above)
Comment (diegofelix) [reply src/Console/ClientCommand.php:99]: Done

File: .travis.yml [hunk 3]
```diff
@@ -10,6 +10,7 @@ services: mongodb
 before_install:
   - if [[ $(phpenv version-name) = "7.1" ]]; then pecl install mongodb; fi
   - echo "extension = mongodb.so" >> ~/.phpenv/versions/$(phpenv version-name)/etc/php.ini
+  - phpenv config-rm xdebug.ini || true
```
Comment (ravanscafi) [inline .travis.yml]: why? without this we will have no coverage metrics

image

**Review by ravanscafi — APPROVED** [review]

**Pull request merged**